client/src/components/ChatForm.tsx

```diff
@@ -0,0 +1,232 @@
+import React, { useState, useEffect } from "react";
+import { createMuiTheme, makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import SendIcon from "@material-ui/icons/Send";
+import { continueSession, SuccessfulContinuationResponse } from "api";
+import { errorForStatus } from "./ErrorConfig";
+
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      main: "#1b6a9c",
+    },
+  },
+});
+
+const useStyles = makeStyles((theme) => ({
+  chatbox: {
+    width: "90%",
+    maxWidth: 400,
+  },
+  button: {
+    margin: theme.spacing(1),
+  },
+}));
+
+export default function ChatForm(props: {
+  lesson: string;
+  messages: { senderId: string; type: string; text: string }[];
```

Comment:
use the explicit types everywhere in this method to streamline and avoid DRY violations